src/Common/Auth/OpenIDConnect/Entities/UserEntity.php

```diff
@@ -31,6 +32,22 @@ public function getClaims()
         $claimsType = (!empty($_REQUEST['grant_type']) && ($_REQUEST['grant_type'] === 'client_credentials')) ? 'client' : 'oidc';
         if ($claimsType === 'oidc') {
             $uuidToUser = new UuidUserAccount($this->identifier);
+            $fhirUser = '';
+            $userRole = $uuidToUser->getUserRole() == 'users';
```

Comment:
maybe my brain isn't working here since it's late, but won't this cause $userRole to equal a boolean (`$userRole = ($uuidToUser->getUserRole() == 'users')`)

Reply:
I need to revisit this code, that should force a boolean and should break, need to figure out why its still working haha

Reply:
Odd, it didn't seem to impact the behavior here at all, not sure why.

Reply:
Fixed it and tested that everything is still working.